assets/scss/components/vsipCalendar.scss

```diff
@@ -0,0 +1,103 @@
+// hide calendar on unsupported browsers
```

Comment:
Old browsers not supported by GOVUK Frontend will basically just get a list of radio options for visit slots.